mysql-to-kv-delta/function.yaml.template

```diff
@@ -0,0 +1,55 @@
+
+# Copyright 2017 The Nuclio Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+spec:
+  description: "This function imports data from MySQL database into Iguazio NoSQL table on an ongoing basis.It requires having create time and modified time columns in order to fetch the delta in each iteration"
+  handler: "main:handler"
+  env:
+    - name: CONTAINER
+      value: {{ .Container }}
+    - name: TABLE
+      value: {{ .Table }}
+    - name: IGZ_V3F
+      value: {{ .IgzV3f }}
+    - name: IGZ_USER
+      value: {{ .IgzUser }}
+    - name: IGZ_PWD
+      value: {{ .IgzPwd }}
+    - name: SQL_HOST
+      value: {{ .SqlHost }}
+    - name: SQL_PORT
+      value: {{ .SqlPort }}
+    - name: SQL_USER
+      value: {{ .SqlUser }}
+    - name: SQL_PWD
+      value: {{ .SqlPwd }}
+    - name: SQL_DB_NAME
+      value: {{ .SqlDBName }}
+    - name: SQL_QUERY
+      value: {{ .SqlQuery }}
+    - name: DELTA_INTERVAL_MINUTE
+      value: {{ .SqlQueryDeltaInterval }}
+    - name: CREATED_DATETIME_COL
+      value: {{ .SqlQueryCreatedDtCol }}
+    - name: MODIFIED_DATETIME_COL
+      value: {{ .SqlQueryModifiedDtCol }}
+  runtime: "python:3.6"
+  minReplicas: 1
+  maxReplicas: 1
+  build:
+    functionSourceCode: {{ .SourceCode }}
+    commands:
+      - pip install v3io-frames sqlalchemy mysqlclient pandas
+    baseImage: "python:3.6"
```

Comment:
please add empty line at end of file (relevant to the rest of the files, it is a standard - to have empty line at EOF)